src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/AvslagAktivitetskravDelregel.java

```diff
@@ -89,9 +99,21 @@
     private Specification<FastsettePeriodeGrunnlag> sjekkOmMorKombinasjonArbeidUtdanning() {
         return rs.hvisRegel(SjekkOmMorKombinasjonArbeidUtdanning.ID, SjekkOmMorKombinasjonArbeidUtdanning.BESKRIVELSE)
                 .hvis(new SjekkOmMorKombinasjonArbeidUtdanning(), morKombinasjonArbeidUtdanning())
+                .ellers(sjekkOmMorOppgittUføretrygd());
+    }
+
+    private Specification<FastsettePeriodeGrunnlag> sjekkOmMorOppgittUføretrygd() {
+        return rs.hvisRegel(SjekkOmMorOppgittUføre.ID, SjekkOmMorOppgittUføre.BESKRIVELSE)
+                .hvis(new SjekkOmMorOppgittUføre(), sjekkOmMorBekreftetUføretrygd())
                 .ellers(Manuellbehandling.opprett("UT1314", null, Manuellbehandlingårsak.MOR_UFØR, true, true));
     }
 
+    private Specification<FastsettePeriodeGrunnlag> sjekkOmMorBekreftetUføretrygd() {
+        return rs.hvisRegel(SjekkOmMorBekreftetUføre.ID, SjekkOmMorBekreftetUføre.BESKRIVELSE)
+                .hvis(new SjekkOmMorBekreftetUføre(), Manuellbehandling.opprett("UT1314", null, Manuellbehandlingårsak.MOR_UFØR, true, true))
```

Comment:
Legge sluttepunktet i egen metode siden det er 2 veier til del? Så slipper vi å ha det definert 2 steder i koden.

.hvis(new SjekkOmMorBekreftetUføre(), ut1314())

Reply:
Done. Tror denne blir kortlivet gitt signaler om avslagsbehandling aktivitet for dette tilfellet